File: 120583.py
"""중복된 숫자 개수"""
def solution(array,n):
    count = 0
    for arr in array:
        if str(n) == str(arr):
            count +=1
    return count

print(solution([1,2,3,4,1],1))

# 다른사람풀이

def solution(array, n):
    return array.count(n)

File: 120583.js
// 중복된 숫자 개수
function solution(array, n) {
  let count = 0;
  for (let i = 0; i < array.length; i++) {
    if (array[i] == n) {
      count++;
    }
  }
  return count;
}
console.log(solution([1, 1, 1, 1, 1, 1], 1));

// 다른사람풀이
function solution(array, n) {
  var answer = 0;
  let Array = array.filter((item) => item === n);
  answer = Array.length;

  return answer;
}
